option optimize_for = LITE_RUNTIME;

message SellAwakenItemRequest
{
 repeated uint32 pos = 1;
}
